test_scripts/RC/OnRCStatus/commonOnRCStatus.lua

```diff
@@ -278,6 +294,12 @@ function m.validateOnRCStatusForHMI(pCountOfRCApps, pExpData, pAllocApp)
         return compareValues(pExpData[AnotherApp], d.params, "params")
       end
     end)
+  :ValidIf(function(_, d)
+    if d.params.allowed then
```

Comment:
@GetmanetsIrina Presence of some argument should be verified with d.payload.allowed ~= nil code